implementations/typescript/src/self-contracts/typescript-kit.self-contracts.test.ts

```
@@ -0,0 +1,890 @@
+}
+
+function mathFloorValue(n: number): number {
+  return Math.floor(n);
+}
+
+it("compute_cid_output_length_eq_139", () => {
+  fc.assert(fc.property(fc.string(), (b) => computeCidLength(b) === 139), { numRuns: 8 });
+});
+
+it("compute_cid_is_deterministic", () => {
+  fc.assert(fc.property(fc.string(), (b) => computeCidForString(b) === computeCidForString(b)), { numRuns: 8 });
```

Biome (2.4.15)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)